server.js

```diff
@@ -28,6 +149,61 @@ app.use(express.json());
 app.get("/", (req, res) => {
   res.send("Hello Technigo!");
 });
+const { Schema } = mongoose;
+const userSchema = new Schema ({
+  name: String,
+  age: Number,
+  alive: Boolean
+});
+
+const User = mongoose.model("User", userSchema);
+
+const songSchema = new Schema({
+    id: Number,
+    trackName: String,
+    artistName: String,
+    genre: String,
+    bpm: Number,
+    energy: Number,
+    danceability: Number,
+    loudness: Number,
+    liveness: Number,
+    valence: Number,
+    length: Number,
+    acousticness: Number,
+    speechiness: Number,
+    popularity: Number
+})
+
+const Song = mongoose.model("Song", songSchema);
+
+
+
+app.get("/songs/id/:id", async (req, res) => {
+  try {
+    const singleSong = await Song.findById(req.params.id);
```

Comment:
For an additional endpoint that gets all of whatever property you would like from your Schema, you could use Song.find instead of findById